src/launchpad/kafka.py

```diff
@@ -205,7 +205,20 @@ class LaunchpadStrategyFactory(ProcessingStrategyFactory[KafkaPayload]):
 
     @staticmethod
     def _initialize_worker_logging() -> None:
-        """Initialize logging in worker process."""
+        """Initialize logging in worker process.
+
+        With multiprocessing spawn context, subprocesses don't inherit
+        parent's stdout/stderr. We need to explicitly configure logging
+        to write to stdout for Docker/GCP log collection.
+        """
+        import sys
```

Comment:
Any reason to not add this top-level (genuine question too, my python knowledge ain't deep enough to know)

Reply:
no, it should be at the top. i'm just lazy and on my 3rd attempt. I have little faith in this but want to try it before doing a more complex approach lol - but ya I'll move it , all g